.github/workflows/ci.yml

```diff
@@ -17,3 +17,5 @@ jobs:
           node-version: ${{ matrix.node-version }}
       - run: npm install
       - run: npm test
+      - run: npm run test:ts
+        if: matrix.node-version == '18.x'
```

Comment:
Since the test no longer uses multiple loaders, can more versions be tested?

Reply:
Since it's using the built-in test runner that is introduced in 18.x, it can only run on 18.x. However I think it's also possible to run the same way as other tests.

Reply:
Looks like the tests are failing on 16.x.

Reply:
I think it requires versions greater than or equal to [16.12.0](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md#experimental-esm-loader-hooks-api).

I'll exclude 16.10.0 from the workflow step.

Reply:
@rochdev updated the PR. Please check again